rules/eslint.js

```diff
@@ -59,9 +71,5 @@ module.exports = {
         message: "`with` is disallowed in strict mode because it makes code impossible to predict and optimize.",
       },
     ],
-
-    // The airbnb config forces single quote, but we prefer double quote
-    // https://eslint.org/docs/latest/rules/quotes
-    quotes: ["error", "double", { avoidEscape: true }],

```

Comment:
why did you remove this?

Reply:
this rule in the eslint gets disabled by the typescript-eslint plugin, I didn't removed it, I moved it to the typescript rule